schemas/deviceToCloud/device/device.json

```diff
@@ -127,12 +128,13 @@
                 },
                 "appName": {
                     "type": "string"
+                },
+                "iccid": {
```

Comment:
https://github.com/nrfconnect/sdk-nrf/blob/3797fac81db0b317d14abded836b3ebc3220979c/lib/modem_info/modem_info_params.c#L39

ICCID, along with IMSI and UICC mode, has been reported in the "simInfo" section for a long time.
again, no preference, but all NCS code should be aligned.